src/main/java/core/basesyntax/shape/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class RightTriangle extends Shape {
+    private double base1;
+    private double base2;
```

Comment:
Rename 'base2' to 'leg2' for better clarity as right triangles have legs, not bases.